tests/utils/scales-utils-tests.js

```diff
@@ -505,6 +505,22 @@ test('scales-utils #getScaleFnFromScaleObject', t => {
     [-1, 1],
     'should build a generic domain that reflects about zero'
   );
+
+  const ordinalScale = getScaleFnFromScaleObject({
```

Comment:
maybe add a test for this that also capture padding? or does this do that already?

Reply:
We are already testing padding of 0.5 (which is padding we use in react-vis).
This is so-called 'outer' padding (padding before first bar and after last bar).
Inner padding is set to 1.0 in d3-scale, as we are really using 'point-scale' from d3-scale.

Reply:
👌